src/node_file.cc

```diff
@@ -1142,6 +1196,41 @@ static void Stat(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+static void StatSync(const FunctionCallbackInfo<Value>& args) {
+  Realm* realm = Realm::GetCurrent(args);
+  BindingData* binding_data = realm->GetBindingData<BindingData>();
+  Environment* env = realm->env();
+
+  CHECK_GE(args.Length(), 3);
+
+  BufferValue path(realm->isolate(), args[0]);
+  bool use_bigint = args[1]->IsTrue();
+  bool do_not_throw_if_no_entry = args[2]->IsFalse();
+  CHECK_NOT_NULL(*path);
+  THROW_IF_INSUFFICIENT_PERMISSIONS(
+      env, permission::PermissionScope::kFileSystemRead, path.ToStringView());
+
+  env->PrintSyncTrace();
```

Comment:
I think this is just for debugging, right?

Reply:
This line was already exist in SyncCall implementation and removal of this line breaks a test.